.github/workflows/ci.yml

```diff
@@ -19,8 +19,34 @@ jobs:
         env:
           GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
 
-  unit-test:
+  autoblack:
     needs: commit-lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
+      - name: Install Black
+        run: pip install black
+      - name: Run black --check .
+        run: black --check .
+      - name: If needed, commit black changes to the pull request
+        if: failure()
+        run: |
+          black .
+          git config --global user.name 'autoblack'
+          git config --global user.email 'maximilianwerk@users.noreply.github.com'
+          echo $GITHUB_HEAD_REF
+          echo $GITHUB_REPOSITORY
+          git remote set-url origin https://x-access-token:${{ secrets.GITHUB_TOKEN }}@github.com/jina-ai/jina
+          git checkout $GITHUB_HEAD_REF
+          git commit -am "style: automatic codeformating with black"
```

Comment:
would it make sense just to ammend to last commit?

Reply:
or it will be disturbing for the user to mess with their commits?

Reply:
I like the idea. I'll test it

Reply:
I am not even sure if this works on a PR. If a user forks our repository, we cannot commit to their fork. So perhaps, it is better to only black on a merge.

Reply:
yes black on a merge I think is the way to go